### ISSUE TITLE
Can Sponge support domain decomposition?

### ISSUE THREAD

**Issue by limin2021** [mindspore-ai/mindscience#1]
I wonder to know if Sponge can support domain decomposition?

**Comment by Xia-Yijie** [comment]
No. Domain decomposition is mainly for CPU parallelism, and SPONGE is not designed to be used on CPUs. The design of parallel computing in CPU, GPU and Ascend is different.
